lib/core_lwt.ml

```diff
@@ -9,7 +9,7 @@ module Std = struct
     include Core_lwt_extra
     module Main = Lwt_main
     module Unix = Lwt_unix
-    module Chan = Lwt_chan
+    module Chan = Lwt_io
```

Comment:
This substitution is not valid, `Chan` is a module alias for `Lwt_chan`, and the same is true for other modules. We just moving here from the pseudo `Lwt_` namespace, to a modular `Lwt.`, e.g., `Lwt_xxx` is now `Lwt.Xxx`.

Reply:
During build without the change:
```
File "lib/core_lwt.ml", line 12, characters 18-26:
Warning 3: deprecated: module Lwt_chan
 Use module Lwt_io.
```
Do you want to rename `Chan` to `Io` for the substitution?

Reply:
We can either drop Chan at all, or keep it deprecated. I think that we can easily just remove it.